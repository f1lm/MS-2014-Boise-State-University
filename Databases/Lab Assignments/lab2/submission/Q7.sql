select FNAME, LNAME, CNAME, LGrade from student S 
inner join minor M  on M.SSN = S.SSN
inner join course C on C.DeptCode = M.DeptCode
inner join grade_report G on G.SSN = S.SSN and G.CourseNo = C.CourseNo;