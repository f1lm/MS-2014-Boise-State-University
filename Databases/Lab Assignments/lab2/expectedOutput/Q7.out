mysql> source Q7.sql;
+---------+----------+-----------------+--------+
| FNAME   | LNAME    | CNAME           | LGrade |
+---------+----------+-----------------+--------+
| Joe     | Ford     | Intro to CS I   | C      |
| Susan   | Johnson  | Digital Systems | B      |
| Alex    | Mace     | Calculas I      | D      |
| Richard | McDonald | Databse Theory  | A      |
+---------+----------+-----------------+--------+
4 rows in set (0.00 sec)

mysql> notee;
